Exercises/unique_token/Solution/CMakeLists.txt

```
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.16)
 project(KokkosTutorialUniqueToken)
-include(../../common.cmake)
 
-add_executable(uniquetoken unique_token.cpp)
-target_link_libraries(uniquetoken Kokkos::kokkos)
+# Add a custom module path for find_package
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake)
 
+find_package(Kokkos REQUIRED)
 
```

```suggestion
if (NOT (Kokkos_DEVICES MATCHES "OPENMP" OR Kokkos_ENABLE_OPENMP))
```

For consistency with the above remark?

---

The solution works with any backend.